HLTrigger/special/plugins/HLTPixelThrustFilter.cc

```diff
@@ -0,0 +1,107 @@
+#include "HLTrigger/HLTcore/interface/HLTFilter.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "DataFormats/HLTReco/interface/TriggerFilterObjectWithRefs.h"
+#include "DataFormats/SiPixelCluster/interface/SiPixelCluster.h"
+#include "RecoLocalTracker/ClusterParameterEstimator/interface/PixelClusterParameterEstimator.h"
+#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
+#include "RecoLocalTracker/Records/interface/TkPixelCPERecord.h"
+#include "PhysicsTools/CandUtils/interface/Thrust.h"
+
+//
+// class declaration
+//
+
+class HLTPixelThrustFilter : public HLTFilter {
```

Comment:
Given that the code does not use the HLTFilter functionality, it is better to avoid the overhead and use a plain EDFilter instead.

Reply:
Changed to standard EDFilter in 31fb30855064b1366f53d5483f7d7402e7c6ddc1